platform/broadcom/saibcm-modules/debian/changelog

```diff
@@ -1,3 +1,9 @@
+opennsl (5.0.0.11) unstable; urgency=medium
```

Comment:
Wondering if this file was changed and included when you built the libsaibcm debians?  If not, this version will not be reflected when one issue bcmcmd "bsv" correct?

Reply:
libsaibcm is itself a debian and it has the version defined there in saibcm 
```
networking-acs-saibcm$ git grep 5.0.0.11   
RELEASE:5.0.0.11
debian/changelog:saibcm (5.0.0.11) unstable; urgency=medium
include/brcm_sai_version.h:#define BRCM_SAI_VER "5.0.0.11"
```
We update this modules file -- so that we can track which saibcm release caused this change.

Reply:
I guess there are no minor involved yet this time so it is probably ok for now... it there are minor involved such as something like 5.0.0.11-1 then the "-1" will be missing if the debian/changelog is not included as part of the debian build...